lib/content/content.go

```diff
@@ -28,9 +31,19 @@ func getProviders(c *configure.Conf) []provider {
 	ret := make([]provider, 0, len(c.Providers()))
 	for _, p := range c.Providers() {
 		for alias, meta := range convertInterfaceToMap(p) {
-			ret = append(ret, provider{alias, meta})
+			m := convertInterfaceToMap(meta)
+			if _, ok := m["priority"]; !ok {
+				m["priority"] = DEFAULT_PRIORITY
+			}
+
+			ret = append(ret, provider{alias, m})
 		}
 	}
+
+	sort.Slice(ret, func(i, j int) bool {
```

Comment:
The amount of calculation `getProviders` is the worst case O(2n^2).
I do not feel well, but I think that there is no problem because the number of `providers` is not large.

Reply:
I think that there is no problem because it is executed only once at startup.